tests/integration/test_kms.snapshot.json

```diff
@@ -496,19 +496,19 @@
     }
   },
   "tests/integration/test_kms.py::TestKMS::test_sign_verify[RSA_2048-RSASSA_PSS_SHA_256]": {
-    "recorded-date": "16-03-2023, 19:16:06",
+    "recorded-date": "12-04-2023, 14:09:57",
     "recorded-content": {
       "signature": {
-        "KeyId": "arn:aws:kms:<region>:111111111111:key/a48829a6-6127-494e-be9a-4a62a4021caa",
-        "Signature": "b'\\n\\x89\\x92\\xef\\xe3\\xd2d\\xfb\\x9a\\xc27\\x04\\xef\\x92e\\xa1\\xc9\\x86\\x919\\xb8\\xd5\\x0f\\x1d\\x1bv\\xe2\\xe5\\x13@=Ql\\x02\\xfeh\\x1dPEX*\\x93\\x05\\xe7\\xde\\xa9\\xa5\\x01\\xc3z#\\x81U\\xf1B\\xf4^\\xc6\\xb9VI\\x988\\xa4l\\x0e\\xb8\\x81\\xf4\\xb7\\xb3D\\xe3:\\xceN\"\\xd9\\xcb\\xf6\\xd6\\x1b\\xaf#\\xdf\\x14yh\\xb9\\xde RrD\\x12\\x1e\\xac\\xdewbkp\\x97\\xe7{__J\\xde\\xf5\\x9c\\xca.\\xadAg\\xe2\\xf3\\x8fPT\\x82\\x14\\x19LoW\\xc5\\xf3\\x7f\\xe3a\\x1a+\\xd6p7\\xd9\\x0c\\xa9\\xf7w\\x10<\\xef\\xdb\\xe1\\xb8\\xbc\\xed\\x82.o\\xd4h\\x1c\\xb6]\\xa20\\xe5V\\x1b\\xc3\\x1c*+r=\\xc4\\x8b\\tXp\\x91!Y^L\\xbaN\\xa7\\x97\\xd0\\xd1R\\xfb\\xd3\\xae`\\xa7\\x993[\\xfa\\xd5\\x8c_J\\xcak\\xab\\xbew\\xa6\\x88\\xb6B\\x90(vG\\x14\\x1b \\r\\xf9\\xdd\\xe0B\\xd5KJ/\\xa2\\xa0[\\r\\xfc\\x97\\xfe\\xb6\\xee0\\x89\\x8e\\x94\\xc4,i\\xc60\\x01]E\\xfd\\xf7\\x93i\\x17ois\\xbeZ&'",
+        "KeyId": "<key-id:1>",
+        "Signature": "b'5\\xab8\\x02\\r\\xb9\\xb1\\xc7H\\tZ[\\xa4%\"\\xc0\\xce\\xc5\\x992\\xa2\\x97\\x04\\x1e\\x85\\xdeu\\x85\\xcd\\x8c\\xfc\\x82\\xc0K\\xa5KY\\x9a\\xd5\\x94\\x13mh-\\xd8\\xb3\\x81V\\xa4\\xa6\\x1d!\\tF\\xd2\\x1a\\xda\\xeb%O\\xf2l\\xf6\\xee-\\x11\\x87\\xa8\\x9a\\xd1\\xd9o\\xfe\\xab\\xeb\\xd7N\\x0eY]!\\x89\\xca\\'.\\xdayx(G\\xf1\\x97\\x13G\\x80!\\x0b\\x06\\x1d\\x1c\\x86\\x12yc\\xd28\\xb1A\\xa9Yg+7X\\x81k\\x8d\\xe1\\xb0\\xc8\\x10\\xf1z\\x8d\\xca\\x057MB;hFt\\x0f!\\xe5\\x01n\\xe4\\xe5  W?\\xf2\\xde^\\xba\\xea\\x10]2\\xf6\\x80SzN\\x13\\x16z\\x07{OVi\\xeb\\xfe)\\xbf\\xc65\\xa3\\xf8\\x90\\xdcJ[\\xa9\\x0c\\x96Z\\xa9\\x1e\\x0f\\x8c\\xa6\\xde\\x9d\\xfb\\xbd\\x04\\xe9\\x81:\\x16\\x01\\xf3\\xa3L\\xe9M\\xb5\\xe3Be\\xad\\xd2\\x8c2Kql\\x8e\\x82\\xfb\\x9f\\x9d\\xc8Q\\xf0\\xd5\\xcd\\x1a\\\\!\\xccY\\xdc\\x03\\x87\\xa1\\xb8\\xf9\\xdeb\\x90\\x9c\\xa4\\xe6\\x9e\\xf2\\x82\\xcc5?\\xbb\\xdcVa\\xc8\\xf2.\\xea\\xdb%d'",
```

Comment:
NIT: as we create new keys, the signature changes for every run and could also be transformed. Currently the field `Signature` is part of the `skip_snapshot_verify`